addons/web_hierarchy/static/src/hierarchy_renderer.js

```diff
@@ -97,12 +97,13 @@ export class HierarchyRenderer extends Component {
     }
 
     get rows() {
-        const rootNodes = this.props.model.root.rootNodes;
+        const rootNodes = this.props.model.root.rootNodes.filter((n) => !n.hidden);
```

Comment:
Strange... can't we use OWL to have a dynamic property and a display none based on that? This feels a bit like "the old ways" ^^

Reply:
Currently the model is not reactive nor used in a reactive object so all re-renders have to be done with `notify`.
In the template there is a logic for rows based on `row_first` and `row_last`, so I'm filtering the records in this getter so that logic is not impacted